schema.sql

```diff
@@ -6,5 +6,9 @@ CREATE TABLE animals (
     date_of_birth DATE NOT NULL,
     escape_attempts INT NOT NULL,
     neutered BOOLEAN NOT NULL,
-    weight_kg DECIMAL NOT NULL
+    weight_kg DECIMAL NOT NULL,
+    COLUMN species VARCHAR(150)
```

Comment:
- Please do not directly/manually add a column in the schema as this is considered bad practice, Just like you altered the table below, that is a more professional way of adding columns to your table. Please fix this 🙏